doc/pedersen_commitments/pedersen-commitments.md

```diff
@@ -0,0 +1,109 @@
+# Pedersen Commitments
+
+In order to obfuscate the amount of energy that is being transferred, the amount of energy is encrypted using Pedersen commitments. The Pedersen commitment is a commitment scheme that allows the holder of the commitment to commit to a value without revealing the value itself. The commitment can be opened to reveal the value. The holder cannot open it to reveal another value. The commitment scheme is homomorphic in the sense that the product of two commitments will be a commitment to the sum of the two values.
+
+## Pedersen commitments are defined as follows:
+$$c=C(m,r)$$
+|identifier|property|usage|accessability|
+|---|---|---|---|
+`c` | commitment | the commitment itself | public |
+`m` | message | the message that is being committed to | private |
+`r` | randomness | the randomness used to commit to the message | private |
+
+```mermaid
+graph TD
+    A[Commitment] --> B[Message]
+    A --> C[Randomness]
+```
+
+### Additional properties of Pedersen commitments
+There are commutation and homomorphism properties that are useful for the Pedersen commitment scheme. These properties are defined as follows:
+
+
+## What these properties can do in practice for Energy Track and Trace
```

Comment:
ProjectOrigin?

Reply:
Yes we are indeed working on project origin.